packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts

```diff
@@ -1,11 +1,8 @@
-export const scrollIntoView = (target: HTMLElement | null | undefined, winArg?: Window | null) => {
+export const scrollIntoView = (target: HTMLElement | null | undefined) => {
```

Comment:
Removing this isn't a problem as this is not part of the package's public API surface.